src/app/admin/moderation/apologies/page.tsx

```
@@ -0,0 +1,127 @@
+    process.env.NEXT_PUBLIC_SUPABASE_URL!,
+    process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY!
+);
+
+export default function AdminApologiesPage() {
+    const [apologies, setApologies] = useState<ApologyRecord[]>([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [filter, setFilter] = useState<'all' | 'pending' | 'approved' | 'rejected'>('all');
+
+    useEffect(() => {
+        async function fetchApologies() {
+            let query = supabase
```

ESLint

[error] Delete `······`

(prettier/prettier)